test/setup/iam.tf

```
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  int_required_roles = [
+    "roles/cloudkms.admin",
+
+    # Needed to run verifications:
+    "roles/owner"
+  ]
+}
+
+resource "google_service_account" "int_test" {
+  project      = module.kms-project.project_id
+  account_id   = "kms-int-test"
+  display_name = "kms-int-test"
+}
+
+resource "google_project_iam_member" "int_test" {
+  count = length(local.int_required_roles)
+
+  project = module.kms-project.project_id
+  role    = "${element(local.int_required_roles, count.index)}"
```

This can be simplified with 0.12.

```suggestion
  role    = local.int_required_roles[count.index] 
```